src/org/opendatakit/briefcase/ui/reused/ButtonProcessing.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.opendatakit.briefcase.ui.reused;
+
+import javax.swing.JButton;
+
+/**
+ * Defines common behavior used in buttons
+ */
+public interface ButtonProcessing {
+
+  void onClick(Runnable callback);
```

Comment:
I believe that `getJButton()` would be the only method that needs to exist in this interface. All the other methods are accessed using the subclasses.

Another method in this interface could be the one I suggested in a previous comment i.e. `void click(Runnable callback)` method.

Reply:
Still have the onClick method in the interface.  DId not fully understand how to implement the change of button.addActionListener(__ ->   to  button.onClick(().
Can you explain how the "__" reference is used? I have not seen that before.

Reply:
My bad! This is something I do when I'm forced to declare an arg that I will ignore in the lambda's body block. `__` is simply the shortest name a variable can have using `_` characters, and the goal would be to implicitly say that the can be ignored.